packages/ui/src/theme/tokens/components/switchField.ts

```diff
@@ -1,11 +1,74 @@
+import {
+  BackgroundColorValue,
+  BorderColorValue,
+  BorderRadiusValue,
+  BoxShadowValue,
+  DesignToken,
+  FontSizeValue,
+  OpacityValue,
+  SpaceValue,
+  TransformValue,
+  TransitionDurationValue,
+} from '../types/designToken';
+
+interface SwitchFieldDisabledTokens {
+  opacity: DesignToken<OpacityValue>;
+}
+
+interface SwitchFieldFocusedTokens {
+  shadow: DesignToken<BoxShadowValue>;
```

Comment:
Should the property be `boxShadow` instead of `shadow`?

Reply:
I agree. But I think change the name here would also modify the CSS variable name?